data/maps/BattleFrontier_Lounge3/scripts.inc

```diff
@@ -134,18 +134,18 @@ BattleFrontier_Lounge3_EventScript_LostChallenge::
 	end
 
 BattleFrontier_Lounge3_EventScript_RewardBet5::
-	buffernumberstring STR_VAR_1, (BET_AMOUNT_5 * 2)
-	setvar VAR_0x8004, (BET_AMOUNT_5 * 2)
+	buffernumberstring STR_VAR_1, BET_AMOUNT_5 * 2
+	setvar VAR_0x8004, BET_AMOUNT_5 * 2
```

Comment:
Parenthesis here are extraneous and when used unnecessarily like this can make automated analysis a bit more finicky

Reply:
Certain compilers will interpret this incorrectly as multiple arguments without the parentheses. Please revert this.